src/style/app.module.css

```diff
@@ -339,15 +348,23 @@
   align-items: center;
 }
 
+.searchButton:hover {
+  background-color: var(--search-button-hover-bg, #005bb5);
```

Comment:
This color seems too dark for the current color scheme. Use a pre-existing color value